compose/ui/ui/src/uikitMain/kotlin/androidx/compose/ui/res/ComposeResources.kt

```diff
@@ -0,0 +1,44 @@
+/*
```

Comment:
By the way, we should name files with ".uikit.kt" suffix (we can rename all files in the separate PR someday).

We do that in androidMain and desktopMain to avoid conflicts (MPP compiler has a flaw, at least it was before, but now we should keep the naming strategy)